pkg/bbgo/session.go

```diff
@@ -23,6 +23,8 @@ import (
 	"github.com/c9s/bbgo/pkg/util"
 )
 
+var KLineLimit int64 = 1000
```

Comment:
Rename it to historyKLineLimit ? and provide a setter `SetHistoryKLineLimit` ?

Reply:
set to KLinePreloadLimit.  
I think no `Set` function should be fine.